packages/collector/test/immediate/test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
```

Comment:
This is the integration test for the change in `@instana/collector`. It starts an application under test via `npm start` and verifies that only the actual application under test is instrumented.